packages/ui/src/components/page/Sidebar/CurrentMember/CurrentMember.tsx

```diff
@@ -1,16 +1,18 @@
 import React, { useState } from 'react'
 import styled from 'styled-components'
-import { ArrowDownExpandedIcon, ArrowIcon } from '../../../../components/icons/ArrowDownExpandedIcon'
-import { MembershipsCount } from '../../../../components/MembershipCount'
 import { BorderRad, Colors, Transitions } from '../../../../constants'
 import { useMembership } from '../../../../hooks/useMembership'
+import { AddMembershipModal } from '../../../../modals/AddMembershipModal'
 import { AddMembershipButton } from '../../../AddMembershipButton'
+import { ArrowDownExpandedIcon, ArrowIcon } from '../../../icons'
 import { MemberDarkHover, MemberInfo } from '../../../MemberInfo'
+import { MembershipsCount } from '../../../MembershipCount'
 import { SwitchMemberModal } from './SwitchMemberModal'
 
 export const CurrentMember = () => {
   const { count, members, active } = useMembership()
   const [isOpen, setIsOpen] = useState(false)
+  const [isCreateOpen, setIsCreateOpen] = useState(false)
```

Comment:
As it's a very common pattern, it's worth extracting a hook:
`const [isOpen, toggleOpen] = useToggle()`